include/io.h

```diff
@@ -5,6 +5,10 @@
 #include <iostream>
 #include <memory>
 #include <sstream>
+#include <vector>
+
+#include <mesh.h>
```

Comment:
These are not `std` includes, and are local includes, use `"mesh.h"` instead `<mesh.h>`